pandas-stubs/_libs/tslibs/timestamps.pyi

```diff
@@ -59,9 +59,6 @@ class Timestamp(datetime):
     def __new__(
         cls,
         ts_input: np.integer | float | str | _date | datetime | np.datetime64 = ...,
-        # Freq is deprecated but is left in to allow code like Timestamp(2000,1,1)
-        # Removing it would make the other arguments position only
-        freq: int | str | BaseOffset | None = ...,
         tz: str | _tzinfo | int | None = ...,
         unit: str | int | None = ...,
```

Comment:
We need to clean this up based on the current API.  `tz` and `unit` should move after `nanosecond`